Library/ViewModels/PostCampaignCheckoutViewModel.swift

```diff
@@ -110,40 +110,23 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
       .map { _ in AppEnvironment.current.currentUser }
       .map(isNotNil)
 
-    let notChangingPaymentMethod = context.map { context in
```

Comment:
One thing you could potentially do is throw in an assert like ``assert(!context.isUpdating)`` to document that it's unexpected behavior (for now).